GameDataParser/Parsers/ItemOptionStaticParser.cs

```diff
@@ -26,95 +26,48 @@ protected override List<ItemOptionStaticMetadata> Parse()
-                                    break;
-                                }
-                                optionsStatic.Stats.Add(new ParserStat(ItemAttribute.AttackSpeed, int.Parse(node.Attributes[item.Name].Value)));
+                                optionsStatic.Stats.Add(new ParserStat(ItemAttribute.AttackSpeed, int.Parse(node.Attributes[item.Name].Value.Split(",").First())));
                                 break;
                             case "atp_value_base":
-                                if (node.Attributes[item.Name].Value.Contains(','))
-                                {
-                                    optionsStatic.Stats.Add(new ParserStat(ItemAttribute.Accuracy, int.Parse(node.Attributes[item.Name].Value.Split(",").First())));
-                                    break;
-                                }
-                                optionsStatic.Stats.Add(new ParserStat(ItemAttribute.Accuracy, int.Parse(node.Attributes[item.Name].Value)));
+                                optionsStatic.Stats.Add(new ParserStat(ItemAttribute.Accuracy, int.Parse(node.Attributes[item.Name].Value.Split(",").First())));
                                 break;
                             case "bap_value_base":
                                 optionsStatic.Stats.Add(new ParserStat(ItemAttribute.BonusAtk, int.Parse(node.Attributes[item.Name].Value)));
                                 break;
                             case "cad_value_base":
-                                if (node.Attributes[item.Name].Value.Contains(','))
-                                {
-                                    optionsStatic.Stats.Add(new ParserStat(ItemAttribute.CriticalDamage, int.Parse(node.Attributes[item.Name].Value.Split(",").First())));
-                                    break;
-                                }
-                                optionsStatic.Stats.Add(new ParserStat(ItemAttribute.CriticalDamage, int.Parse(node.Attributes[item.Name].Value)));
+                                optionsStatic.Stats.Add(new ParserStat(ItemAttribute.CriticalDamage, int.Parse(node.Attributes[item.Name].Value.Split(",").First())));
                                 break;
                             case "cap_value_base":
-                                if (node.Attributes[item.Name].Value.Contains(','))
-                                {
-                                    optionsStatic.Stats.Add(new ParserStat(ItemAttribute.CriticalRate, int.Parse(node.Attributes[item.Name].Value.Split(",").First())));
-                                    break;
-                                }
-                                optionsStatic.Stats.Add(new ParserStat(ItemAttribute.CriticalRate, int.Parse(node.Attributes[item.Name].Value)));
+                                optionsStatic.Stats.Add(new ParserStat(ItemAttribute.CriticalRate, int.Parse(node.Attributes[item.Name].Value.Split(",").First())));
```

Comment:
What happen then if is not contains the ',' ?

Reply:
The entire thing becomes a 1-element array